src/main/java/me/contaria/seedqueue/gui/wall/SeedQueueWallScreen.java

```diff
@@ -870,4 +875,11 @@ private static ClientWorld getWorld(WorldRenderer worldRenderer) {
         return ((WorldRendererAccessor) worldRenderer).seedQueue$getWorld();
     }
 
+    @Override
+    public void removed() {
+        super.removed();
+        if (this.client != null) {
+            this.client.getToastManager().clear();
+        }
+    }
```

Comment:
Codestyle nitpicks:
- remove the `super.removed()` call, the parent method is empty
- replace the if statement with `assert this.client != null`, I'm not a big fan of assert usually but `Screen#client` being annotated as `@Nullable` is only because it's nullable before the screen is initialized and i think an `assert this.client != null` at the top of a method signals more clearly that we know `this.client` won't actually be null

Reply:
> Codestyle nitpicks:
> 
> * remove the `super.removed()` call, the parent method is empty
> * replace the if statement with `assert this.client != null`, I'm not a big fan of assert usually but `Screen#client` being annotated as `@Nullable` is only because it's nullable before the screen is initialized and i think an `assert this.client != null` at the top of a method signals more clearly that we know `this.client` won't actually be null

thanks for pointing these out, done (: